draft-ietf-webtrans-http3.md

```diff
@@ -231,56 +232,55 @@ type, followed by the session ID, encoded as a variable-length integer, followed
 by the user-specified stream data ({{fig-unidi}}).
 
 ~~~~~~~~~~ drawing
-  0                   1                   2                   3
-  0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                           0x54 (i)                          ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                        Session ID (i)                       ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-  |                         Stream Body                         ...
-  +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+Stream Type (i) = 0x54,
+Session ID (i),
+Stream Body (..),
 ~~~~~~~~~~
 {: #fig-unidi title="Unidirectional WebTransport stream format"}
 
 ## Bidirectional Streams
 
-WebTransport endpoints can initiate bidirectional streams by opening an HTTP/3
-bidirectional stream and then immediately sending a special signal value 0x41,
-followed by the associated session ID, both encoded as a variable-length
-integer; the rest of the stream is the application payload of the WebTransport
-stream ({{fig-bidi-client}}).
+All client-initiated bidirectional streams are reserved by HTTP/3 as request
+streams, which are a sequence of HTTP/3 frames with a variety of rules; see
+{{Sections 4.1 and 6.1 of HTTP3}}.
+
+WebTransport extends HTTP/3 to allow clients to declare and use alternative
+request stream rules.  Once WebTransport has been established
+({{establishing}}), a client can send a special signal value, encoded as a
+variable-length integer, as the first bytes of the stream in order to indicate
+how the remaining bytes on the stream are used.
+
+WebTransport extends HTTP/3 by defining rules for all server-initiated
+bidirectional streams.  Once WebTransport has been established
+({{establishing}}), a server can open a bidirectional stream and SHALL send a
+special signal value, encoded as a variable-length integer, as the first bytes
+of the stream in order to indicate how the remaining bytes on the stream are
+used.
+
+TODO: consider extracting special signal values into a separate draft;
```

Comment:
Kind of sad to add a TODO here vs. just having an issue for it?

Reply:
(Just if already fixing conflicts)

Reply:
This is effectively just porting the old TODO that existed into this. I agree it could actually be an issue, or just removed if we don't want to do it (as Victor suggested, making a standalone HTTP extension with no context seems a bit problematic)

Reply:
Fair enough 😃